src/index.css

```diff
@@ -1,3 +1,273 @@
+  padding-right: 5rem;
+  line-height: 7px;
+  border-left: 1px solid #e8e8e8;
+}
+
+.percentageVal {
+  margin: 0;
+  font-size: 2rem;
+  color: #121212;
+}
+
+.completed {
+  opacity: 0.5;
+  font-size: 0.875rem;
+  color: #121212;
+}
+
+.books-list-section li > .secondCol > li:nth-child(2) {
+  padding-right: 6rem;
+  padding-left: 0.5rem;
+}
+
+.lastCol {
+  font-family: 'RobotoSlab', sans-serif;
+}
+
+.lastCol .currChapter {
+  opacity: 0.5;
+  font-size: 0.813rem;
+  font-weight: 300;
+  color: #121212;
+}
+
+.lastCol .chapter {
+  font-size: 1rem;
+  font-weight: 300;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
+
+.lastCol > button {
+  font-size: 0.813rem;
+  font-weight: 300;
+  border: none;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+  border-radius: 3px;
+  background: var(--main-acent-color);
+  padding: 5px 12px;
+  width: max-content;
+}
+
+.navigation > ul > li > a {
+  text-decoration: none;
+}
+
+.add-book {
+  justify-content: space-between;
+}
+
+.navigation ul > *:hover {
+  cursor: pointer;
+}
+
+.navigation > ul > li:nth-child(2) a {
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.navigation > ul > li:nth-child(3) a {
+  opacity: 0.5;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.navigation > ul > li:nth-child(1) h1 a {
+  font-size: 1.875rem;
+  color: var(--main-acent-color);
+  text-decoration: none;
+}
+
+.category {
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  color: #121212;
+  margin: 0;
+}
+
+.title {
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+  color: #121212;
+  margin: 0;
+}
+
+.Author {
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+  margin: 0 0 1rem 0;
+}
+
+.firsCol-btns button {
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  background: transparent;
+  border: none;
+  color: #4386bf;
+  padding: 0 10px 0 10px;
+  border-right: 1px solid #e8e8e8;
+  cursor: pointer;
+}
+
+.firsCol-btns button:last-child {
+  border: none;
+}
+
+.firsCol-btns button:first-child {
+  padding-left: 0;
+}
+
+.user-profile-wrapper {
+  justify-content: center;
+  align-items: center;
+  border: solid 1px #e8e8e8;
+  border-radius: 50%;
+  padding: 10px;
+}
+
+.add-book-section {
+  flex-direction: column;
+  padding-bottom: 100px;
+}
+
+.books {
+  width: 80%;
+  margin: 0 auto;
+}
+
+.hr {
+  background: #e8e8e8;
+  height: 1px;
+  margin: 35px 0 15px 0;
+}
+
+.add-book-section h1 {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+:is(input, select) {
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  padding: 12px;
+  width: 55%;
+}
+
+:is(input, select):focus {
+  outline: 3px solid #0290ff;
+}
+
+select {
+  width: 20% !important;
+}
+
```

Comment:
- [ ] Kindly add some margins between the input fields so that it matches that of Zeplin design.
Your design
![Capture](https://user-images.githubusercontent.com/85095702/166829384-f0285ba6-0a2c-4ca0-857b-a9793f7b383e.PNG)
Zeplin design
![zeplin](https://user-images.githubusercontent.com/85095702/166829489-495855ed-9e9e-45a7-955a-b9c5faf0a77c.PNG)